src/main/java/org/italiangrid/storm/webdav/authz/VOMSAuthenticationFilter.java

```diff
@@ -50,9 +50,9 @@ protected Object getPreAuthenticatedPrincipal(HttpServletRequest request) {
 
   public Object extractPrincipal(X509Certificate cert) {
 
-    return cert.getSubjectDN().getName();
+    return cert.getSubjectX500Principal().getName();
```

Comment:
I know that we should use X500 format so I'm in favor of this.
Just to understand this better: what's the reasons behind this change?